model/stories.go

```diff
@@ -1,20 +1,37 @@
 package model
 
-type Story struct {
-	ID     int    `json:"id"`
-	Title  string `json:"title"`
-	Author string `json:"author"`
-}
+import (
+	"gorm.io/gorm"
+	"time"
+)
 
-var stories = []Story{
-	{ID: 1, Title: "Story 1", Author: "Username1"},
-	{ID: 2, Title: "Story 2", Author: "Username2"},
+var DB *gorm.DB
+
+type Story struct {
+	StoryID      int       `json:"story_id"`
+	UserID       int       `json:"user_id"`
```

Comment:
Would it be better if this is called "Author ID" instead of "User ID"? This will make the purpose of this field clearer.